tensorflow/compiler/mlir/tosa/transforms/legalize_utils.h

```diff
@@ -283,11 +283,11 @@ inline bool IsTFLDoubleRoundingMode() {
 #endif  // TFLITE_SINGLE_ROUNDING
 }
 
+Value reshapeScalarTo1D(PatternRewriter& rewriter, Location loc, Value value);
```

Comment:
why moved this function?

Reply:
There was a merge conflict between `reshapeScalarTo1D` and `broadcastLowRankTensor`. It has been resolved by aligning the order of these two functions in both the source and header files.